v4/source/system.yaml

```
@@ -349,6 +349,47 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/config/patch':
+    put:
+      tags:
+        - system
+      summary: Patch configuration
+      description: |
+        Submit configuration to patch. As of server version 4.8, the `PluginSettings.EnableUploads` setting cannot be modified by this endpoint.
+        ##### Permissions
+        Must have `manage_system` permission.
```

Please document the minimum server version
```suggestion
        Must have `manage_system` permission.
        __Minimum server version__: 5.20
```